Good-Bids/goodbids
# [#94] Metrics but WP

# Summary

- Adds driver and metrics component
- Adjusts Tailwind content paths
- Adds inputs for new components
- Adds script to handle components in dev mode

## Issues

- #94

## Testing Instructions

- Start WP (`bin/start`) and Vite (`bin/start-vite`)
- Also start webpack (`cd client-mu-plugins/goobids && npm run start`) (working on this)
- Create an auction with the `sample bidding` block
- Visit said auction
- See the metrics block

## Screenshots

![image](https://github.com/Good-Bids/goodbids/assets/48295174/5c3a045e-0aae-4619-b8f0-e838e095eb2f)
